### PR TITLE
feat: added typography component

### PR DESCRIPTION
**Pull request by krate27**
https://user-images.githubusercontent.com/86555869/160281130-c9bbb454-8918-407d-904a-7f7fe782f0b6.mp4

### CODE REVIEW

File: Components/Alert.css [hunk 1]
```diff
@@ -37,17 +37,17 @@
     display: flex;
     flex-direction: column;
     gap: 15px;
-    border: 3px solid var(--light-gray);
+    border: 1px solid black;
```
Comment (abhisheksahani-nova) [inline Components/Alert.css:40]: Use css variables

**Review by abhisheksahani-nova — APPROVED** [review]
Clean and subtle code looks good to merge

File: Components/Alert.css (hunk 1 above)
Comment (OmKakatkar) [inline Components/Alert.css:40]: For being consistent try to use css variables for colors like it is seen in the rest of your code

**Review by OmKakatkar — APPROVED** [review]
Good to see CSS variables for Colors.
You can also add variables for Font Sizes and Border properties as well, it can be done later if required.
Looks good

**Review by i-am-a-codegirl — APPROVED** [review]
Code is clean and looks good to merge

**Comment by Syedmha** [comment]
Good use of CSS variables and simple code. Reviewed.

**Pull request merged**